packages/widget-library/src/Node.vue

```diff
@@ -9,7 +9,7 @@
 
       <span class="node version" v-if="routable">
         <router-link
-          :to="{ name: 'reader', query: { urn: metadata.firstPassageUrn } }"
+          :to="{ name: 'reader', params: { urn: metadata.firstPassageUrn }, query }"
```

Comment:
@paltman In my opinion, I think we should either remove the querystring here or introduce a `$scaife.config` option for including it / excluding it at the site level.

I think if I'm navigating within a particular text (ancestors / siblings / pagination), I'd expect the display mode to carry over, but it feels like a bit of a chore if I go to the library widget and change to another text to then have to also change the display mode.

What do you think?

Reply:
@jacobwegner are you saying for the library you don't want to preserve the display mode but have it go back to default, or at least want it being a config setting?

Reply:
@paltman Yes; I think a site-level config (something like `$scaife.config.widgetLibrary.includeQuery`), and default to `includeQuery: true`.

I can think of widgets in Digital Sira or within Scholarly Editions where we'd want this type of control, so it would be good to demonstrate this pattern.

As I said above, I don't think we need it _everywhere_ we create links to the `reader` route, but for widgets like this where we're going to be changing the _text_ being viewed (not just the _passage_ within a text/version), I think it is a reasonable option to have.

Reply:
just dropping it for now